sdk/search/search-documents/src/searchIndexClient.ts

```diff
@@ -138,16 +138,18 @@ export class SearchIndexClient {
   }
 
   /**
-   * Retrieves a list of existing indexes in the service.
-   * @param options Options to the list index operation.
+   * Retrieves a list of existing SearchIndexes in the service.
```

Comment:
the concept should still be called `indexes` right?

Reply:
After discussion with Ramya, reverted all changes to method and parameter names.